docs/v2/azure/supported-resources/deprecated.md

```diff
@@ -0,0 +1 @@
+These resource versions are deprecated and will be removed in an upcoming ASO release. Migration to newer versions is advised. See [Breaking Changes](https://azure.github.io/azure-service-operator/guide/breaking-changes/) for more information.
```

Comment:
Where are these files displayed? They seem pretty bare.

Reply:
Or are they just injected into the documentation but not actually displayed themselves? If so that makes sense.

Reply:
This directory contains fragments that are injected into the generated files where specified.